### DISCUSSION TITLE
Proxmox installation accessing photos on NAS

### DISCUSSION THREAD

**Discussion by ajocius** [photoprism/photoprism#4240] [General]
I have Photoprism installed on Proxmox using script from helper-scripts. I can open web interface, but can not ssh to it's using IP address (port 22) and same user name and psw as to web ui. Does it require different password for ssh connection? is there default ssh user name and psw? I can import pictures using web ui import functionality. However I do not want to save pictures on my Proxmox pc memory. I would like to keep them on NAS, and link Photoprism to that location. I have tried to mount following, hoping that I will be able to see pictures:
pct set 107 -mp0 /media/DNS323/Photo,mp=/originals
pct set 107 -mp0 /media/DNS323/Photo,mp=/import
However my library is empty (besides one picture that I manually imported as test). Same NAS is mounted for other apps like Frigate and Jellyfin, so I think my Proxmox has required access to my NAS.
How do link Photoprism to picture library on NAS (by the way, it is very simple storage device that I access via IP address and mapped network drive on windows)?
How do I ssh into Photoprism in case I need to make adjustments to some config files there?

**Comment by scblakely** [comment]
`I have Photoprism installed on Proxmox using script from helper-scripts. I can open web interface, but can not ssh to it's using IP address (port 22) and same user name and psw as to web ui.`

You log in to the LXC via ssh with **root** and the password you configured during setup.
